spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
+dynamically. Further, the publisher of the files placed on the web can use
+capabilities like [CDNs] to distribute the files.
+
+Likewise, an active component might be used by the controller of the DID to
+automate the process of publishing and updating the DIDDoc and KEL.
 
 #### Read (Resolve)
 
 Before a `did:webs` DID can undergo standard resolution, it must be converted
-back to a DDURL. This is done by replacing the "did:webs:" prefix with
-https://", converting a percent-encoded port delimiter back to a colon, and
-replacing all other colons with slashes. In addition, if the DID includes no
-path, then "/.well-known/did.json" is appended.
-
-An HTTP GET is now performed on the DDURL to fetch a DID doc. This operation
-MUST follow both temporary and permanent redirects, but MUST break redirect
-cycles and MUST NOT allow excessive redirects. The final URL once all redirects
-have been followed is called the [[ref: terminal URL]]. If GETting the terminal
-URL returns HTTP status code 404, if the terminal url contains a path, and if
-the terminal url does not end in ".json", then a client MUST append ".json" to
-the terminal URL and make a second attempt to GET the resource with the revised
-URL.
-
-Unlike `did:web`, the resolution process is not assumed to produce valid output
-simply because a DID doc is returned. The top-level `id` property in the doc
-contains a KERI AID rather than the value of the DID itself. DID doc is
-analyzed, and If the DID is a A second GET MUST also be performed. This one
-fetches the KEL. The URL for the KEL is generated as follows:
-
-1.  Start with the final URL that successfully fetched a DID doc after all
-    redirects were followed.
-2.  If this URL ends with ".json", change it to end with ".kel.json". Otherwise,
-    just append ".kel".
-
-When a DID is converted back into a DDURL for resolution to a DID doc,
-
-TODO: what about OOBIs? TODO: what about resolution against a previous version
-TODO: what args are valid in DID URL? TODO: service endpoints TODO: security and
-privacy considerations, from did:web TODO: read did:web DNS sections
+back to an HTTPS URL [as described above](#method-specific-identifier). The
+reader then does a GET on both the URL for the DIDDoc (ending in `/did.json`)
+and on the URL for the KEL (ending in `/did.kel`)
 
-#### Update
+On receipt of the two documents, the KEL is processed using [KERI Rules] to
+verify the validity of the KEL itself, and of the DIDDoc.
 
-This operation is not possible if the AID uses [[ref: direct mode]]. Otherwise:
+#### Update
 
-1.  Modify the AID's key state or its [[ref: witnesses]] (resulting in an updated
-    [[ref: KEL]], and/or its transaction history (resulting in an updated
-    [[ref: TEL]]), using KERI mechanisms. This may involve notifying witnesses.
-2.  Re-derive the DID doc.
-3.  Republish the KEL and the TEL on the website.
+Updates are made to the AID by adding KERI events to the [[ref: KEL]]. Some of
```

Comment:
I know you said you don't want to support KERI AIDs that use direct mode. I'm not so sure we should preclude this. Maybe we don't tout it, but I'm tempted to imply support for it by changing this sentence to read, "If an AID is updateable, updates are made..."

Reply:
OK changed, but…I still haven’t seen a definition of what a direct mode KERI AID is… :-).